historical_system_profiles/listener_logging.py

```diff
@@ -31,7 +31,9 @@ def config_cloudwatch(logger):
 
 def initialize_logging():
     kafkalogger = logging.getLogger("kafka")
-    kafkalogger.setLevel("ERROR")
+    kafkalogger.addHandler(logging.StreamHandler(sys.stdout))
+    kafkalogger.setLevel("DEBUG")
```

Comment:
Should this be set by `LOG_LEVEL` env var?

Reply:
Yes, it does! Thanks